quizler/lib.py

```diff
@@ -14,14 +15,40 @@ def get_api_envs():
     return client_id, user_id
 
 
-def api_call(end_point, client_id, user_id):
+def api_call(method: str, end_point: str, params: Dict[str, str] = None, client_id: str = None,
+             access_token: str = None):
```

Comment:
Should not we drop type annotation altogether? The only help they provide is extra highlighting for me in PyCharm and I suspect that's not enough value to keep them. By the way, it won't work with Python 2 and it may be useful to port `quizler` to it later.

Reply:
I think sometimes the information is valuable, so I have now moved this information to the docstrings.